samples: nrf9160: http_update: graceful shutdown of modem fw (#6429)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Added support for minimal assistance using factory almanac, time and location.

> * nRF9160: HTTP update samples:
> 
>   * HTTP update samples will now send the modem to power off mode after the firmware image download is complete. This will avoid ungraceful disconnects from the network upon pressing reset button on kit.

Reviewer: ```suggestion
* nRF9160: HTTP update samples:

  * HTTP update samples now set the modem in the power off mode after the firmware image download completes. This avoids ungraceful disconnects from the network upon pressing the reset button on the kit.
```
